### PR TITLE
Add WiredTiger holds open many handles playbook page

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

File: DhandleCloseIdle.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# WiredTiger holds open many handles
```
Comment (agorrod) [inline DhandleCloseIdle.md]: I'd structure this more like:

Poorer performance than expected with WiredTiger in a workload with many databases, collections and/or indexes

File: DhandleCloseIdle.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Problem
+
+In a system with a large number of collections + indexes access times on tables get slower and slower over time. One factor that influences this is large number of open, but unused dhandles.
```
Comment (agorrod) [inline DhandleCloseIdle.md]: I think it's worth clarifying the relationship between collections/indexes and tables here. i.e: that collections and indexes are MongoDB concepts layered on top of WiredTiger tables.

I've seen a number of different problems due to large numbers of data handles, including:
* Running out of file descriptors
* Stalls or slow operations due to instance wide operations (checkpoint, fsyncLock, etc)
* General performance lower than expected

I'd also be inclined to split this topic up for users who actively use a lot of handles, and users who have a lot of idle handles. We have a solution for the latter, but the former is still a work in progress.

**Comment by daveh86** [comment]
@agorrod can you take a look at this?

**Comment by agorrod** [comment]
Thanks Dave - I've made a few little edits. Please go ahead and merge if you are happy with those.

**Pull request merged**